### PR TITLE
Update JWK public key format in protocol

### LINKED ISSUE #107: KBS attestation protocol shows invalid JWK
The key shown in the [KBS attestation protocol documentation](https://github.com/confidential-containers/kbs/blob/5d94595e08849d75e4d99f23b417fce4b5d5243b/docs/kbs_attestation_protocol.md#public-key) isn't a valid JWK.
See https://github.com/confidential-containers/guest-components/issues/184

### PR DESCRIPTION
**Pull request by katexochen**
Fixes #107

### CODE REVIEW

File: docs/kbs_attestation_protocol.md [hunk 1]
```diff
@@ -288,7 +289,7 @@ The authentication service is provided by the KBS through two endpoints:
 2. `/kbs/v0/attest` only accepts `POST` requests whose body is a [KBS Attestation](#attestation)
    JSON payload and the header contains a `Cookie` set to the value received in
    step 1.i. This is how the attester replies to attestation challenge received
-   in step 1.ii. If the attestation was successful, the KBS replies to that request with an 
+   in step 1.ii. If the attestation was successful, the KBS replies to that request with an
```
Comment (sameo) [inline docs/kbs_attestation_protocol.md:292]: You have a few cosmetic/formatting changes that are unrelated to this PR. Could you please keep those aside so that we dont mix two different set of change in the same PR?

Comment (katexochen) [reply docs/kbs_attestation_protocol.md]: Sorry, my bad.

**Comment by katexochen** [comment]
This requires https://github.com/confidential-containers/attestation-service/pull/119 to be merged first, and the `as-types` and `attestation-service` version to be updated.

Also requires https://github.com/confidential-containers/guest-components/pull/177 to be merged first, and `kbs_protocol` version to be updated.
